starter-code/css/main.css

```diff
@@ -6,3 +6,158 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+.nav {
+    background-color: white;
+    width: 100vw;
+    height: 60px;
+    top: 12px;
```

Comment:
`top: 0` sticks it to the top of the page. otherwise you notice the background just above the nav bar as you scroll.